app/static/js/request/main.js

```diff
@@ -155,6 +155,11 @@ var customRequestFormData = {}; // tracks the data of custom forms that will be
 var formCategories = {}; // tracks what category each form belongs to
 var categorized = false; // determines if selected agency has categorized forms
 var currentCategory = ""; // tracks the current category that can be submitted for the selected agency
+var categorySelected = false; // a flag that determines if a category has been set yet or not
+var categoryDividerText = "────────────────────";
+var defaultCateogryInfoText = "Note: This agency has categorized the different types of requests a user can submit and they are separated in the dropdown below. Your request may have multiple submissions of only one category.";
+var defaultCategoryWarningText = "Selecting this option will remove any existing information for other request types. Are you sure you want to change categories?";
```

Comment:
There is no business owner now that NYPD doesn't want forms. We can decide on language as a team unless you want Pauline to weigh in. This is just what I used in the meantime. I assumed each agency will want different language in this section anyway so I added functionality to pull custom language.

Reply:
I think we should have Pauline or Ken double check the language (at least our defaults). I believe they had asked for that during the last meeting with Law / Mayor's Office.

Reply:
Checking the language can happen after this is merged, since it is a simple fix. Just wanted to flag it for follow-up. I'll add a comment to the Jira ticket for Martin to follow up